### ISSUE TITLE
Nikon D3500 compatibility report

### ISSUE THREAD

**Issue by danwolff** [dognotdog/ptpwebcam#19]
# D3500 with PTP Webcam v1.0.0-alpha8

Tested on macOS Catalina 10.15.5 

## TL;DR
Seems to work quite well, have not noticed any bugs yet but time will tell.

## Overall
 - Seems to work in Quicktime
 - Seems to work in Bluejeans desktop app
 - Initially seemed to not be a visible source when testing at [cam-recorder.com](https://www.cam-recorder.com/ ) in Chrome, or in Chrome for the another web VC tool my employer uses a lot, but this turned out to be the [known issue](https://github.com/dognotdog/ptpwebcam#known-issues) about signing; the recommended `codesign` actions for Chrome have granted access to these web apps (at least for now).

## Details from adjusting the menu items
 - Initially, attempting to change White Balance did not appear to do anything (stayed on Automatic setting) at first, but then after adjusting Exposure Program Mode (e.g., to Manual) the White Balance setting did appear to make changes.  This is probably as anticipated, just noting it.
 - Adjusting Exposure Program Mode does appear to make the anticipated changes, with visible output changes.
 - With Exposure Program Mode set to Manual, ISO adjustment may have made changes but it was hard for me to tell.
 - With Exposure Program Mode set to Manual or Automatic, Exposure Correction does appear to make the anticipated changes to the output.
 - Autofocus seems to work pretty much as expected, tested in Manual and Automatic Exposure Program modes.  It seems likely to be the focus settings configured in the camera, but this was not tested extensively.
 - "LiveView Status (On)"" appears in the PTP Webcam menu and appears grayed out.  I assume this is anticipated, not sure what it is supposed to do but it could be a confirmation of connectivity.

## General usage notes
 - The back of the camera is black when an application uses the D3500 source, then switches to the settings display (as configured) when an application using it is closed.  This seems good, I certainly would not want to mirror the output to the camera's display which is not facing me in the VC setup (for energy, heat, wear, etc.).
 - The D3500 was also working since earlier v1.0.0-alpha releases, but just getting to this summary now.  Overall it seems to work!  Glad this exists, thanks a ton.  After more extended testing, e.g. as a VC daily driver, there could be more refined notes.

**Comment by dognotdog** [comment]
@danwolff fantastic report, thank you! The settings UI could definitely use more work, eg. see #24, as the ones visible in the control menu are only the very basics at this point, and other camera specific settings might prevent some other settings from taking effect as expected.
